gcsfilesystem/src/test/scala/cromwell/gcsfilesystem/GoogleConfigurationSpec.scala

```diff
@@ -0,0 +1,161 @@
+package cromwell.gcsfilesystem
```

Comment:
I was going to complain that the same `google` block is now used by both `engine` and `gcsfilesystem` but... if it's ok with @mcovarr then it's ok with me!